### DISCUSSION TITLE
(Postgrex.Error) FATAL 28P01 (invalid_password) password authentication failed for user "postgres"

### DISCUSSION THREAD

**Discussion by ueukxvj** [plausible/analytics#3496] [Self-Hosted Support]
All i changed is
plausible-conf.env
BASE_URL=http://plausible.company.com
SECRET_KEY_BASE=JFpSyvEihrFhajsiizvkJplWHm7CE2PzuKnDfNoV0xoF1p29QopZHxEDzJ4oqfA53na+LKlZwv3043TRp0HwDQ==

Then im trying **docker compose up** and thats what i see:

docadm@docker:/docker/plausible/hosting$ docker compose up
[+] Running 4/3
 ✔ Container hosting-plausible_events_db-1  Created                                                                   0.1s
 ✔ Container hosting-mail-1                 Created                                                                   0.1s
 ✔ Container hosting-plausible_db-1         Created                                                                   0.1s
 ✔ Container hosting-plausible-1            Created                                                                   0.1s
Attaching to hosting-mail-1, hosting-plausible-1, hosting-plausible_db-1, hosting-plausible_events_db-1
hosting-plausible_db-1         |
hosting-plausible_db-1         | PostgreSQL Database directory appears to contain a database; Skipping initialization
hosting-plausible_db-1         |
hosting-plausible_db-1         | 2023-11-07 14:56:08.804 UTC [1] LOG:  starting PostgreSQL 14.9 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r10) 12.2.1 20220924, 64-bit
hosting-plausible_db-1         | 2023-11-07 14:56:08.804 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
hosting-plausible_db-1         | 2023-11-07 14:56:08.804 UTC [1] LOG:  listening on IPv6 address "::", port 5432
hosting-plausible_db-1         | 2023-11-07 14:56:08.814 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
hosting-plausible_db-1         | 2023-11-07 14:56:08.822 UTC [22] LOG:  database system was shut down at 2023-11-07 14:52:29 UTC
hosting-plausible_db-1         | 2023-11-07 14:56:08.831 UTC [1] LOG:  database system is ready to accept connections
hosting-plausible_events_db-1  | Processing configuration file '/etc/clickhouse-server/config.xml'.
hosting-plausible_events_db-1  | Merging configuration file '/etc/clickhouse-server/config.d/docker_related_config.xml'.
hosting-plausible_events_db-1  | Merging configuration file '/etc/clickhouse-server/config.d/logging.xml'.
hosting-plausible_events_db-1  | Logging warning to /var/log/clickhouse-server/clickhouse-server.log
hosting-plausible_events_db-1  | Logging errors to /var/log/clickhouse-server/clickhouse-server.err.log
hosting-mail-1                 |     1 LOG: MAIN
hosting-mail-1                 |     1   exim 4.89 daemon started: pid=1, -q30m, listening for SMTP on port 25 (IPv6 and IPv4)
hosting-plausible_events_db-1  | 2023.11.07 14:56:09.388390 [ 1 ] {} <Warning> Application: Listen [::]:8123 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 23.3.7.5 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
hosting-plausible_events_db-1  | 2023.11.07 14:56:09.388715 [ 1 ] {} <Warning> Application: Listen [::]:9000 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 23.3.7.5 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
hosting-plausible_events_db-1  | 2023.11.07 14:56:09.388987 [ 1 ] {} <Warning> Application: Listen [::]:9004 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 23.3.7.5 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
hosting-plausible_events_db-1  | 2023.11.07 14:56:09.389227 [ 1 ] {} <Warning> Application: Listen [::]:9005 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 23.3.7.5 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
hosting-plausible_events_db-1  | 2023.11.07 14:56:09.593705 [ 1 ] {} <Warning> Application: Listen [::]:9009 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 23.3.7.5 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
hosting-plausible-1            | Loading plausible..
hosting-plausible-1            | Starting dependencies..
hosting-plausible-1            | Starting repos..
hosting-plausible-1            | create Plausible.Repo database if it doesn't exist
hosting-plausible_db-1         | 2023-11-07 14:56:20.522 UTC [31] FATAL:  password authentication failed for user "postgres"
hosting-plausible_db-1         | 2023-11-07 14:56:20.522 UTC [31] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
hosting-plausible_db-1         | 2023-11-07 14:56:20.524 UTC [30] FATAL:  password authentication failed for user "postgres"
hosting-plausible_db-1         | 2023-11-07 14:56:20.524 UTC [30] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
hosting-plausible_db-1         | 2023-11-07 14:56:20.541 UTC [29] FATAL:  password authentication failed for user "postgres"
hosting-plausible_db-1         | 2023-11-07 14:56:20.541 UTC [29] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
hosting-plausible-1            | 14:56:20.572 [error] Postgrex.Protocol (#PID<0.183.0>) failed to connect: ** (Postgrex.Error) FATAL 28P01 (invalid_password) password authentication failed for user "postgres"
hosting-plausible-1            | 14:56:20.573 [error] Postgrex.Protocol (#PID<0.163.0>) failed to connect: ** (Postgrex.Error) FATAL 28P01 (invalid_password) password authentication failed for user "postgres"
hosting-plausible-1            | 14:56:20.573 [error] Postgrex.Protocol (#PID<0.164.0>) failed to connect: ** (Postgrex.Error) FATAL 28P01 (invalid_password) password authentication failed for user "postgres"
hosting-plausible_db-1         | 2023-11-07 14:56:20.629 UTC [32] FATAL:  password authentication failed for user "postgres"
hosting-plausible_db-1         | 2023-11-07 14:56:20.629 UTC [32] DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
hosting-plausible-1            | 14:56:20.630 [error] Postgrex.Protocol (#PID<0.187.0>) failed to connect: ** (Postgrex.Error) FATAL 28P01 (invalid_password) password authentication failed for user "postgres"
hosting-plausible-1            | ** (MatchError) no match of right hand side value: {:error, "killed"}
hosting-plausible-1            |     (plausible 0.0.1) lib/plausible_release.ex:40: anonymous fn/2 in Plausible.Release.createdb/0
hosting-plausible-1            |     (elixir 1.14.3) lib/enum.ex:2468: Enum."-reduce/3-lists^foldl/2-0-"/3
hosting-plausible-1            |     (plausible 0.0.1) lib/plausible_release.ex:39: Plausible.Release.createdb/0
hosting-plausible-1            |     nofile:1: (file)
hosting-plausible-1            |     (stdlib 4.2) erl_eval.erl:748: :erl_eval.do_apply/7
hosting-plausible-1 exited with code 0
^CGracefully stopping... (press Ctrl+C again to force)
Aborting on container exit...

**Comment by ruslandoga** [comment]
👋 @ueukxvj 

Please post your `docker-compose.yml` and describe your setup in more detail, if possible :)

**Comment by ueukxvj** [comment]


**Reply by ruslandoga** [reply]
Right now there is no actionable info for me to help you so I still need more details about your setup.
